openapi/src/main/java/io/micronaut/openapi/visitor/Utils.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2017-2022 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.openapi.visitor;
+
+import java.io.File;
+import java.io.InputStream;
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.Future;
+
+import io.micronaut.core.annotation.AnnotationValue;
+import io.micronaut.core.util.CollectionUtils;
+import io.micronaut.http.server.types.files.FileCustomizableResponseType;
+import io.micronaut.inject.ast.ClassElement;
+import io.swagger.v3.oas.models.security.SecurityRequirement;
+
+import org.reactivestreams.Publisher;
+
+/**
+ * Some util methods.
+ */
+public class Utils {
```

Comment:
Few small suggestions:

- add `@since` tag please. I guess this will go in a 4.4.0 version of this module.
- this class should be final.
- it should have a private empty default constructor